code/game/objects/items/weapons/surgery_tools.dm

```diff
@@ -52,7 +52,7 @@
  */
 /obj/item/cautery
 	name = "cautery"
-	desc = "This stops bleeding."
+	desc = "A surgical tool for preventing bleeding or closing incisions."
```

Comment:
technically i think it stops bleeding, it doesn't prevent it? i mean i guess it prevents further bleeding